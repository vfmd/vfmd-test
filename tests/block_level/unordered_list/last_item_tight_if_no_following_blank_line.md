 * One
 * Two

 * Three

 * Four
 + Next list
