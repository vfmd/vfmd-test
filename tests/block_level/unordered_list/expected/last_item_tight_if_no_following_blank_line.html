<ul>
<li>One</li>
<li>Two</li>
<li><p>Three</p></li>
<li>Four</li>
</ul>
<ul>
<li>Next list</li>
</ul>
